torchbenchmark/models/BERT_pytorch/bert_pytorch/dataset/dataset.py

```diff
@@ -16,7 +16,8 @@ def __init__(self, corpus_path, vocab, seq_len, encoding="utf-8", corpus_lines=N
         self.corpus_path = corpus_path
         self.encoding = encoding
 
-        with open(corpus_path, "r", encoding=encoding) as f:
+        with generator as f:
+        #with open(corpus_path, "r", encoding=encoding) as f:
             if self.corpus_lines is None and not on_memory:
                 for _ in tqdm.tqdm(f, desc="Loading Dataset", total=corpus_lines, disable=QUIET):
```

Comment:
Sorry I don't quite get this line and line 122. On line 122, `on_memory` seems to be `False` and `copus_lines=50000`. Therefore, both this and the [next](https://github.com/pytorch/benchmark/blob/c60023d39556ee1d7e2bddde346309068626e02e/torchbenchmark/models/BERT_pytorch/bert_pytorch/dataset/dataset.py#L25) code block will be skipped. Is this the correct behavior?

Reply:
Like above, on_memory is supposed to be true.